### DISCUSSION TITLE
함께 챌린지 완주하실 분들을 찾습니다! 👋

### DISCUSSION THREAD

**Discussion by devmhl** [devrel-kr/HackaLearn#35] [9 잡담방]
미션 진행과정 공유하고 모르는 거 물어보면서 함께 챌린지 완주하실 분들 있으신가요!!
이제 막 챌린지 1번부터 시작했는데 이거 같이 공부하면 더 재밌을 거 같네요!
누구나 들어와서 오픈챗 할 수 있는 디스코드 서버 공유합니다 https://discord.gg/9rq3csDy
저는 개인으로 참가하지만 개인/팀 참여자 모두모두 환영합니다ㅎㅎ 👍

**Comment by justinyoo** [comment]
멋지십니다! 화이팅! 👍

**Reply by loreenkim** [reply]
오 이런 글 멋지네요. 응원합니다~! :)

**Reply by ingHHK** [reply]
놀러가볼께요~!

**Comment by minhoryang** [comment]
질문은 이 [Discussions](https://github.com/devrel-kr/HackaLearn/discussions)에서 해주셔야, 더 빠른 응답을 받으실 수 있습니다.
그리고 다른 참가자분들에게도 도움이 될 거에요. 화이팅입니다 :)

**Comment by Eomhyunjun** [comment]
늦게 봤는데 초대 코드가 만료되었네요!

**Reply by jung-youjin** [reply]
@Eomhyunjun https://discord.gg/ETgP7ZSZ 👈🏼👈🏼 이 링크는 잘 뜨나요?!?!

**Reply by Eomhyunjun** [reply]
@jung-youjin  넵 감사합니다!!
